libs/fsm.c

```diff
@@ -1043,6 +1043,16 @@ int fsm_init(char *module)
 		return 0;
 	}
 
+
+#ifdef HAVE__ICETRANSNOLISTEN
+	_IceTransNoListen("tcp");
+#else
+	fvwm_debug(__func__, "[%s][fsm_init] "
+		   "_IceTransNoListen is not available. "
+		   "Cannot stop listening for TCP connections.\n", module_name);
+#endif
+
+
```

Comment:
Skip the else block entirely, please.

Reply:
I have removed the output completely now. I do want to point out though that without `_IceTransNoListen`, FVWM is accepting **remote** connections.  Anyone with a network route to the machine can, in the very least, crash FvwmButtons. Unless I am missing something here, this does look like a security risk to me.

Reply:
Actually, no. It's not a big security risk. 

I will merge this.